lib/src/cli/install/install_command.dart

```diff
@@ -85,13 +78,15 @@ class InstallCommand extends Command<void> {
   Future<bool> shouldSkipInstall(Pubspec realmPubspec) async {
     final pubspecFile = await File("pubspec.yaml").readAsString();
     final projectPubspec = Pubspec.parse(pubspecFile);
-    if (PackageNames.all.contains(projectPubspec.name)) {
-      print('Running install command inside ${projectPubspec.name} package which is the development package for Realm. '
+
+    if (Flavor.values.map((f) => f.packageName).contains(projectPubspec.name)) {
+      print(//
+          'Running install command inside ${projectPubspec.name} package which is the development package for Realm.\n'
           'Skipping download as it is expected that you build the packages manually.');
       return true;
     }
 
-    if (realmPubspec.publishTo == 'none') {
+    if (realmPubspec.publishTo == 'none' && !debug) {
```

Comment:
Do we really need this check? What if we remove it in order to be able to test with path dependency before the release?

Reply:
We already have the debug flag. Why not use it here?